sessions/workshops/dive-into-object-oriented-python.md

```diff
@@ -0,0 +1,20 @@
+title: Dive Into Object-oriented Python
+speaker: leonardo-giordani
+---
+Each language has its own object-oriented implementation, that can differ in subtle or unexpected ways from others.
+
+Newcomers to Python - whether they are coming from another language, or learning programming through Python for the first time - sometimes encounter some ‘strange’ issues, but understanding Python’s OOP implementation will help make many of them seem a lot less strange.
+
+This tutorial will introduce beginners to Python’s beautiful but sometimes peculiar implementation of OOP concepts. It’s ideal for people who have a bit of Python knowledge and experience, and need to move from first steps to a deeper understanding.
+
+Please bring your own laptop, as the workshop includes hands-on exercises. The minimum setup is a running Python console (preferably Python 3, but Python 2 is also OK) and your editor of choice.
+
+The recommended setup is a running Jupyter Notebook installation (http://jupyter.org/).
+
+There will be no time to install software during the workshop, so please ensure that everything is already working for you. Run this example program from Dive Into Python 3 to check that all’s in order: http://www.diveintopython3.net/your-first-python-program.html
+
+You can use this code to get a working Jupyter Notebook environment:
+
+virtualenv venv3 -p python3
```

Comment:
Not sure where in the process, but we seem to have lost a small amount of Markdown formatting here.